src/invidious/videos/parser.cr

```diff
@@ -47,6 +55,8 @@ def parse_related_video(related : JSON::Any) : Hash(String, JSON::Any)?
     "view_count"       => JSON::Any.new(view_count || "0"),
     "short_view_count" => JSON::Any.new(short_view_count || "0"),
     "author_verified"  => JSON::Any.new(author_verified),
+    "published"        => JSON::Any.new(published || ""),
+    "publishedText"    => JSON::Any.new(published_time_text || ""),
```

Comment:
Please remove that: internally, we shouldn't store any "published" text, especially since the Youtube side is only in english.

Reply:
Got it. Sorry!